packages/components/src/__actions__/Button/v3/_docs/Button.stories.tsx

```diff
@@ -18,81 +19,123 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const Playground: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+export const Playground: Story = {}
+
+export const ButtonVariants: Story = {
+  render: args => (
+    <>
+      <Button {...args} variant="primary" />
+      <Button {...args} variant="secondary" />
+      <Button {...args} variant="tertiary" />
+    </>
   ),
+  decorators: [
+    Story => (
+      <div className="flex gap-8">
+        <Story />
+      </div>
+    ),
+  ],
 }
 
-export const ButtonWithIcon: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+export const ButtonVariantsReversed: Story = {
+  render: args => (
+    <ReversedColors isReversed={true}>
+      <Button {...args} variant="primary" />
+      <Button {...args} variant="secondary" />
+      <Button {...args} variant="tertiary" />
+    </ReversedColors>
   ),
-  args: {
-    children: (
-      <>
-        Label
-        <AddIcon role="presentation" />
-      </>
-    ),
+  parameters: {
+    reverseColors: true,
   },
+  decorators: [
+    Story => (
+      <div className="flex gap-8 bg-purple-700 p-16">
+        <Story />
+      </div>
+    ),
+  ],
 }
 
-export const ButtonWithIconStart: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+export const ButtonSizes: Story = {
+  render: args => (
+    <>
+      <Button {...args} size="small" />
+      <Button {...args} size="medium" />
+      <Button {...args} size="large" />
+    </>
   ),
-  args: {
-    children: (
-      <>
-        <TrashIcon role="presentation" />
-        Label
-      </>
+  decorators: [
+    Story => (
+      <div className="[&>*]:ms-8">
```

Comment:
Oh no what is this strange looking selector 😅 Are you just wanting some gaps?

Reply:
Fixed with flex